docs/source/topics/con_about-usage-data-collection.adoc

```diff
@@ -0,0 +1,12 @@
+[id="about-usage-data-collection_{context}"]
+= About usage data collection
+
+{prod} prompts you before use for optional, anonymous usage data collection to assist with development.
```

Comment:
Note that this is intentionally stated in a vague way ("CodeReady Containers prompts you before use") -- this is due to the fact that the way you are prompted can vary a great deal between operating systems. Rather than describe each method, I opted to use a generic statement.

Reply:
Agree.